changelogs/fragments/add_commit_confirm_to_rm.yaml

```diff
@@ -0,0 +1,5 @@
+---
+minor_changes:
```

Comment:
can we please update the changelog entry to be more descriptive, this breaks it into 3 list items.